app/src/main/java/com/asfoundation/wallet/wallet_validation/generic/WalletValidationActivity.kt

```diff
@@ -22,14 +22,23 @@ class WalletValidationActivity : BaseActivity(),
   private var minFrame = 0
```

Comment:
This validation method should update the string resources as well.
Please refactor the code to handle different view entries in a generic way